### DISCUSSION TITLE
Where are Java examples

### DISCUSSION THREAD

**Discussion by nagkumar** [nitrictech/nitric#639] [Q&A] [answered]
In the v1 I don't see any Java or spring examples can any pl refer me to that code  examples

**Comment by jyecusch** [comment, marked as answer]
Hi @nagkumar unfortunately the JVM SDK hasn't been updated for compatibility with Nitric v1 yet. Node, Python, Go and Dart have all been updated. We're open to contributions to update the JVM SDK. v1 compatible JVM code can also be generated from the Protobuf contracts.

The core team still intends to update the JVM SDK, but we've been prioritizing the SDKs based on community use and it fell lower on the priority order.
